src/components/AssignContainerDialog/AssignContainerDialog.tsx

```diff
@@ -132,58 +127,57 @@ const AssignContainerDialog: React.FC<AssignContainerDialogProps> = props => {
             endAdornment: loading && <CircularProgress size={16} />
           }}
         />
-        <FormSpacer />
-        <div
-          className={scrollableDialogClasses.scrollArea}
-          style={{ height: containerHeight }}
+      </DialogContent>
+      <DialogContent
+        className={scrollableDialogClasses.scrollArea}
+        id={scrollableTargetId}
+      >
+        <InfiniteScroll
+          dataLength={containers?.length}
+          next={onFetchMore}
+          hasMore={hasMore}
+          scrollThreshold="100px"
```

Comment:
threshold went from `10` to `100`, intended?

Reply:
same here